lib/views/reviews-view.js

```diff
@@ -644,4 +648,23 @@ export default class ReviewsView extends React.Component {
 
     return {lineNumber, positionText};
   }
+
+  /* istanbul ignore next */
+  scrollToThread(threadID) {
+    const threadHolder = this.threadHolders.get(threadID);
+    if (threadHolder) {
+      threadHolder.map(element => {
+        element.scrollIntoViewIfNeeded();
+        return null; // shh, eslint
+      });
+    }
+  }
+
+  async resolveUnresolveThread(thread) {
```

Comment:
We keep hitting situations where we need to name methods that toggle something without having a good word for the toggling action. There isn't really a good alternative for this:

* `resolveOrUnresolveThread()`
* `toggleThreadResolution()`
* ... ?